common/src/main/java/org/broadleafcommerce/common/util/BroadleafMergeResourceBundleMessageSource.java

```diff
@@ -31,7 +31,7 @@
 
 
 /**
- * <p>Replaces the deprecated {@link BLResourceBundleMessageSource} by using a {@link ReloadableResourceBundleMessageSource}
+ * <p>Replaces the deprecated BLResourceBundleMessageSource by using a {@link ReloadableResourceBundleMessageSource}
```

Comment:
Which test? Could you explain what do u mean?

Reply:
*text

Reply:
Oh ok. I thought it would be useful info for someone if they were still using the old class. I can re-word it and say `Replaces the older BLResourceBundleMessageSource` instead.